cfgov/unprocessed/css/header.less

```diff
@@ -51,10 +51,11 @@
     width: 190px;
     height: 40px;
     margin: 10px;
+    margin-top: 0;
```

Comment:
This could be updated to `margin: 0 10px 10px;` to avoid resetting the top margin.